src/actions/prListAction.js

```diff
@@ -0,0 +1,24 @@
+import { FETCH_PRLIST } from './actionTypes';
+import axios from 'axios';
+
+const apiUrl = 'http://localhost:5010/tracker/list';
```

Comment:
base url for backend should be in env file. So add environment file with sample.env